components/scroller/scroller_default.story.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <div>
+    <button @click="scrollToItem(100)">
+      Go to index 100
+    </button>
+    <button @click="scrollToItem(0)">
+      Go to start
+    </button>
+    <button @click="scrollToItem(items.length - 1)">
+      Go to end
+    </button>
+
+    <dt-scroller
```

Comment:
I think we may have to use the [dynamic scroller implementation](https://github.com/Akryum/vue-virtual-scroller/blob/master/packages/vue-virtual-scroller/README.md#dynamicscroller). For the feed list I don't think we are going to know sizes in advance.